crates/sui-graphql-client/src/lib.rs

```diff
@@ -322,8 +323,8 @@ impl Client {
     pub async fn coins(
         &self,
         owner: Address,
-        after: Option<&str>,
-        before: Option<&str>,
+        after: Option<String>,
+        before: Option<String>,
```

Comment:
why require an owned string now?

Reply:
I think I did like this because I need to pass the event filter in the streams calls, so I needed to clone it. Maybe I can just pass a reference.

Reply:
@bmwill this is what I came up with, what do you think?